metron-platform/README.md

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 
 # Current Build
 
-The latest build of metron-platform is 0.4.1.
+The latest build of metron-platform is 0.4.2.
 
```

Comment:
Is this statement still valid?  Just as an aside

Reply:
The above statement is validly updated to reflect the new upcoming version, which is the point of this PR.  However, the file metron-platform/README.md as a whole doesn't make much sense any more.  I've opened https://issues.apache.org/jira/browse/METRON-1271 to address it.